music_assistant/providers/opensubsonic/sonic_provider.py

```diff
@@ -711,11 +711,14 @@ async def get_stream_details(
         )
 
     async def _report_playback_started(self, item_id: str) -> None:
+        self.logger.debug("scrobble for now playing called for %s", item_id)
         await self._run_async(self._conn.scrobble, sid=item_id, submission=False)
 
     async def on_streamed(self, streamdetails: StreamDetails, seconds_streamed: int) -> None:
         """Handle callback when an item completed streaming."""
+        self.logger.debug("on_streamed called for %s", streamdetails.item_id)
```

Comment:
note that this will be called when the buffer is filled - its not perse the actual playback time, especially not for players that buffer a lot of data ahead such as sonos.

If you want actual played seconds we also have an event called by the queue manager.